code/controllers/subsystem/server_maint.dm

```diff
@@ -21,3 +23,6 @@ SUBSYSTEM_DEF(server_maint)
 
 	if(config.sql_enabled)
 		sql_poll_population()
+		if(config.use_exp_tracking)
+			if(REALTIMEOFDAY > (triggertime +3000))	//server maint fires once at roundstart then once every 10 minutes. a 5 min check skips the first fire
```

Comment:
the realtimeofday macro is not working properly and breaks on midnight rollover causing negative inits, what impact will that have on this